packages/components/src/alignment-matrix-control/utils.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export const GRID = [
+	[ 'top left', 'top center', 'top right' ],
+	[ 'center left', 'center center', 'center right' ],
+	[ 'bottom left', 'bottom center', 'bottom right' ],
+];
+
+// Stored as map as i18n __() only accepts strings (not variables)
+export const ALIGNMENT_LABEL = {
+	'top left': __( 'Top Left' ),
+	'top center': __( 'Top Center' ),
+	'top right': __( 'Top Right' ),
+	'center left': __( 'Center Left' ),
+	'center center': __( 'Center Center' ),
+	'center right': __( 'Center Right' ),
+	'bottom left': __( 'Bottom Left' ),
+	'bottom center': __( 'Bottom Center' ),
+	'bottom right': __( 'Bottom Right' ),
+};
+
+// Transforms GRID into a flat Array of values
+export const ALIGNMENTS = GRID.reduce( ( alignments, row ) => {
+	row.forEach( ( cell ) => {
+		alignments.push( cell );
+	} );
```

Comment:
This could be simplified a bit:
```suggestion
	for ( const cell of row ) {
		alignments.push( cell );
	}
```